src/app/shared/components/line-item/line-item-edit-dialog/line-item-edit-dialog.component.html

```diff
@@ -26,9 +26,13 @@
       <!-- input (quantity) -->
       <div class="form-group">
         <div class="row">
-          <label class="col-form-label col" translate>product.quantity.label</label>
+          <ish-product-quantity-label
+            for="line-item-edit-quantity"
+            translationKey="product.quantity.label"
```

Comment:
translationKey is not necessary if it is the default translation key

Reply:
the translation key has been removed